docker/Dockerfile-fargo-master

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.6
```

Comment:
Fargo should work with go version 1.7 but since we are testing all our go software using 1.6 it makes more sense  to stick with that version. I agree that we could use newer version for just testing fargo, but I don't see any benefits from running it on version 1.7.